### DISCUSSION TITLE
My github action has been disabled. Moreover, it appears that no one from GitHub Support has responded to me.

### DISCUSSION THREAD

**Discussion by ovo-Tim** [community/community#135534] [Actions]
### Select Topic Area

Question

### Body

I have no idea about what is wrong with [this action](https://github.com/ovo-Tim/FurLang_server/blob/main/.github/workflows/build.yaml), I don't believe I've done anything against the rules. But even after my friend ran this action, his github action also been disabled. I have been troubled for nearly a week. I have tried to post a [ticket](https://support.github.com/ticket/personal/0/2927236) to github support, but it seems there is noboby answer me. I'm anxious to build my programmes by github action. What should I do?

**Comment by dane** [comment]
Have you tried running your action locally with [`nektos/act`](https://nektosact.com/introduction.html)? I'd start here, see if you're running into any errors locally, and post updates in this thread so we can help.

**Reply by ovo-Tim** [reply]
The test is still in progress. However, the Linux build has been successful, and the tiny bug on windows has been fixed.
![image](https://github.com/user-attachments/assets/fd03f047-cb0b-485f-bafd-63918d6aebbb)

(I don't believe those errors can disable my GitHub Actions workflow; there must be another issue at play.)

**Reply by ovo-Tim** [reply]
I can't get "Act" to work on my computer. However, as I mentioned earlier, I don't believe that the issues with my actions could cause my entire GitHub Actions workflow to be disabled.

**Reply by ovo-Tim** [reply]
@dane :eyes:

**Reply by dane** [reply]
I'm sorry for the delay. Life got a bit crazy. If the windows build is failing, act wouldn't help there since it runs in a docker container, simulating the Linux build. I don't have suggestions on debugging the windows build, unfortunately.

**Reply by ovo-Tim** [reply]
> I'm sorry for the delay. Life got a bit crazy. If the windows build is failing, act wouldn't help there since it runs in a docker container, simulating the Linux build. I don't have suggestions on debugging the windows build, unfortunately.

@dane It seems that you didn't understand what I meant. What I mean is, failures with a specific GitHub Action shouldn't result in the entire GitHub Action being banned.

**Comment by HAMZA-420** [comment]


**Reply by ovo-Tim** [reply]
None of those suggestions are useful. :cry:

**Reply by ovo-Tim** [reply]
> Can you mark my this answer as complete. Let me check other solutions..Thanks

Why?

**Reply by HAMZA-420** [reply]
It will add my contribution. Thanks

**Reply by ovo-Tim** [reply]
> It will add my contribution. Thanks

But I still needs help. I can't simply mark it as complete unless you help me find out the solution. :cat:

**Reply by ovo-Tim** [reply]
> Its solution one then next will be uploaded.

I don't think the "solutions" that generate by LLM can help.

**Reply by kyu08** [reply]
@ovo-Tim 
This account [posted a answer to the discussion I created as well](https://github.com/orgs/community/discussions/136165#discussioncomment-10369885) that is not useful and seems like generated by AI.

According to [the code of conduct of community/community](https://github.com/community/community/blob/fc052e3e356d7fe1df0f92529e9c36be933541f4/CODE_OF_CONDUCT.md), we may able to regard these posts as violation of these terms if these answers are generated by AI.

> - Take personal responsibility for everything you post.
> - Use your expertise as a developer to verify that the answer works and makes sense.

**Reply by ovo-Tim** [reply]
@kyu08 
Is @rikeshdahal also a bot?

**Reply by kyu08** [reply]
I don't know. But that account [posted to mine](https://github.com/orgs/community/discussions/136165#discussioncomment-10369909) too.

If you think the account violates the code of conduct, you can report them.

**Comment by rikeshdahal** [comment]
This is good?

**Reply by ovo-Tim** [reply]
What?

**Comment by HAMZA-420** [comment]


**Reply by ovo-Tim** [reply]
I don't think the "solutions" that generate by LLM can help.

**Reply by LiteBrite82** [reply]
Hi @HAMZA-420 , 

We’ve clarified our stance on using generative AI tools like ChatGPT within our Community via [this announcement](https://github.com/orgs/community/discussions/69006). Please review the guidelines to ensure your post meets them as failure to adhere to those rules can result in action taken by our moderator team.  You can read our updated [Code of Conduct](https://github.com/community/community/blob/baa5cd9020ef3f2bbf86dca1191d9c90f9b0822d/CODE_OF_CONDUCT.md#reasonable-use-of-ai-generated-content) and the [announcement](https://github.com/orgs/community/discussions/69006) for more details. Thank you for helping us maintain an authentic and beneficial space for everyone.

**Comment by ovo-Tim** [comment]
@github Hey, I think there needs a github staff member.

**Comment by ovo-Tim** [comment]
Github support has helped me solve this issues.
